frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -30,34 +37,44 @@ const ApartmentCard = ({ buildingData, numReviews, company }: Props): ReactEleme
 
   return (
     <Card className={classes.card}>
-      <CardMedia className={classes.img} image={img} component="img" title={name} />
-      <CardContent>
-        <Grid container spacing={1}>
-          <Grid container item justify="space-between" alignItems="center">
-            <Grid item>
-              <Grid container alignItems="center">
-                <Typography variant="h6" className={classes.nameText}>
+      <Grid container direction="row">
+        <Grid item md={2}>
+          <CardMedia className={classes.img} image={img} component="img" title={name} />
+        </Grid>
+        <Grid item md={10}>
+          <CardContent>
+            <Grid container spacing={1}>
+              <Grid item>
+                <Typography variant="h5" className={classes.nameText}>
                   {name}
                 </Typography>
               </Grid>
-            </Grid>
-          </Grid>
-          {company && (
-            <Grid container item justify="space-between">
+
+              {company && (
+                <Grid container item justify="space-between">
+                  <Grid>
+                    <Typography variant="subtitle1">{buildingData.address}</Typography>
+                  </Grid>
+                </Grid>
+              )}
+
               <Grid>
```

Comment:
What if we take `HeartRating` out of `Typography`? It'll probably be easier to style them if they're separate. If you add `alignItems` to the parent container, it should vertically align the text with the hearts. 
```
  <Grid container direction="row" alignItems="center">
    <HeartRating value={3} readOnly />
    <Typography variant="h6" className={classes.reviewNum}>
      {numReviews + (numReviews !== 1 ? ' Reviews' : ' Review')}
    </Typography>
  </Grid>
```

Also, you could try adding a `marginLeft` to the styling on the text so that it's not so close to the hearts, kinda like this: 
<img width="946" alt="Screen Shot 2022-03-20 at 5 45 18 PM" src="https://user-images.githubusercontent.com/57203639/159187467-4fb2d0ae-8d6d-4c2e-899f-8b09c96e48d0.png">